Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,515 @@
+Nozzle design is very difficult for the necessary size for the CGT, about 1 inch in length and 0.5 inches in diameter. Boundary layer effects are dominent. To accurately design them
+on a computer would include high level numerical smulation which beyond the team's skill set. If there are recurring problems here, Dr.\ Olles can be consulted as he has experience with this.
+There may also be the need for assistance during the data analysis phase or if the team chooses to perform flow simulations. But it will be a small amount of help.
+
+If assistance is needed for the data analysis, Dr.\ Olles would be a good professor to ask. He has a PhD in rocket science
+so he could certainly help.
+\subsection{Long-Term Vision}
+\label{sec:vision}
+
+The long term vision of this project is to be a building block to the next phase of space propulsion research. CREST, a mission that was proposed in the
+spring of 2016 by James Parkus, relies on a space propulsion system. It is the reason the SPEX propulsion subsystem was formed. Thus, it is natural for the
+CGT to assist in the furthering of this project. The main conern for CREST was the propulsion necessary to de-orbit quickly enough that SPEX can determine its entry location and therefore estimate its landing location.
+
+This project will also be great for
+the members to put on their resume. At the end of this project a detailed document will be made which outlines the most important aspects of the project.
+Then the RCS will build off the momentum created by this project. And the understanding of space propulsion inside SPEX will continue to grow and flourish.
+
+Once the CGT is complete, the team can move onto the next phase of space propulsion research. This is the RCS\@. Reaction control systems are used very
+frequently in space propulsion and attitude adjustment systems as well as rockets. Building an RCS is difficult. But if the team
+has prior experience with cold gas propulsion, their job becomes significantly easier. The team can refer back to their data for the determination of the
+appropriate nozzle characteristics.
+
+After the culmination of the cold gas thruster project, the team can try to design, fabricate, and test an aerospike nozzle, only if time permits.
+There are benefits to aerospikes but there are also disadvantages. The advantages are concerned with the region of optimal expansion. The aerospike nozzle can match the
+ambient pressure over a larger region and therefore can produce more thrust. However, the disadvantages are increased thermal loading and high design and fabrication complexity\cite{SPAD}.
+
+\section*{Acknowledgements}
+The author would like to thank Dr. Garrick and Dr. Olles for their help in understanding the theory of nozzles and the review of the system before testing.
+
+
+\onecolumn
+\appendices{}
+\section{\textbf{Detailed Fall Timeline}}
+\centering
+\begin{tabular}{ c l }
```

Comment:
This table looks a lot better! I recommend you take a look at the package `booktabs` next time you go about making tables, it helps a lot.